packages/react/src/constants.tsx

```diff
@@ -16,3 +16,6 @@ export const Colors = [
 ] as const
 
 export const Gradients = ['blue-purple', 'green-blue', 'pink-blue', 'purple-red', 'red-orange'] as const
+
+export const BaseSizeScale = [4, 8, 12, 16, 20, 24, 28, 32, 36, 40, 44, 48, 64, 80, 96, 112, 128] as const
```

Comment:
Love this. I think we should generate this using style dictionary and serve from our brand-primitives package in the long term, as it's tightly coupled to a dependency. Could we add a FIXME or TODO above this line to indicate this?